lib/backend/k8s/syncer.go

```diff
@@ -46,7 +48,7 @@ type kubeSyncer struct {
 	kc         KubeClient
 	callbacks  api.SyncerCallbacks
 	OneShot    bool
-	tracker    map[model.Key]interface{}
+	tracker    map[string]model.KVPair
```

Comment:
To do that I'd need to marshal the keys as json for the map keys, so we can unmarshal and delete them below.

At a minimum, just store the keys.